bin/setup

```diff
@@ -26,6 +26,9 @@ Dir.chdir APP_ROOT do
   puts "== Copying application.yml =="
   run "cp config/application.yml.example config/application.yml"
 
+  puts "== Copying logstash.conf =="
+  run "cp logstash.conf.example logstash.conf"
```

Comment:
```ruby
  run "cat logstash.conf.example | sed 's/path_to_repo/#{APP_ROOT.to_s.gsub('/', '\/')}/g' > logstash.conf"
```

(I also removed the brackets around `path_to_repo` because they made things harder